cmd/wallet-js-sdk/src/gnap/client.js

```diff
@@ -57,19 +60,18 @@ export class Client {
   async continue(req) {
     // TODO : Implement http message signature module in JS (https://github.com/trustbloc/agent-sdk/issues/348)
     // const sig = this.signer.Sign(req);
+    const url = this.gnapAuthServerURL + AUTH_CONTINUE_PATH;
 
-    const gnapResp = await axios.post(
-      this.gnapAuthServerURL + AUTH_CONTINUE_PATH,
-      req,
-      {
-        headers: {
-          "Content-Type": CONTENT_TYPE,
-          // "Signature-Input": "TODO", // TODO update signature input
-          // TODO : Implement http message signature module in JS (https://github.com/trustbloc/agent-sdk/issues/348)
-          // Signature: encode(sig, true),
-        },
-      }
-    );
+    const gnapResp = await axios.post(url, req, {
+      headers: {
+        "Content-Type": CONTENT_TYPE,
+        "@target-uri": url,
```

Comment:
target uri doesn't need to be a header - the http-signature algorithm is meant to grab the target URI of the request itself.